suite-native/module-dev-utils/src/screens/DemoScreen.tsx

```diff
@@ -267,28 +266,17 @@ export const DemoScreen = () => {
                             <Radio
                                 key="first"
                                 value="first"
+                                accessibilityLabel="First"
                                 onPress={handleRadioPress}
                                 isChecked={radioChecked === 'first'}
                             />
                             <Radio
                                 key="second"
                                 value="second"
+                                accessibilityLabel="First"
```

Comment:
```suggestion
                                accessibilityLabel="Second"
```

Reply:
Yeah, I just copy pasted this and then thought it's just demo screen and doesn't matter too much so didn't want to make new commit just for this :D 

But I can commit your suggestion, thanks